src/utils.js.flow

```diff
@@ -20,17 +20,20 @@ declare export function generatePath(
 
 declare export function findRouterContext(
   location: Location,
-  routes: Routes
+  routes: Routes,
```

Comment:
Never noticed this declaration duplication and it is bad. Let's remove this utils definitions from index.flow, put the correct one here and then re-export this utils types in index

Reply:
Moved the duplicate declarations to util.js.flow and imported them back to index.js.flow.